server/routes/findInterventions/searchResultsPresenter.ts

```diff
@@ -7,7 +7,7 @@ export default class SearchResultsPresenter {
   constructor(
     private readonly interventions: Intervention[],
     private readonly filter: InterventionsFilter,
-    private readonly pccRegions: PCCRegion[]
```

Comment:
should these classes now have both region and location?

Reply:
Rather than PCC Region and NPS region?